zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -278,17 +280,120 @@ def channels_to_zerver_stream(slack_dir: str, realm_id: int, added_users: AddedU
     print('######### IMPORTING STREAMS FINISHED #########\n')
     return zerver_defaultstream, zerver_stream, added_channels, zerver_subscription, zerver_recipient
 
+def channel_message_to_zerver_message(constants: List[Any], channel: str,
+                                      added_users: AddedUsersT, added_channels: AddedChannelsT,
+                                      zerver_userprofile: List[ZerverFieldsT],
+                                      zerver_subscription: List[ZerverFieldsT],
+                                      ids: List[int]) -> Tuple[List[ZerverFieldsT],
+                                                               List[ZerverFieldsT]]:
+    """
+    Returns:
+    1. zerver_message, which is a list of the messages
+    2. zerver_usermessage, which is a list of the usermessages
+    """
+    slack_dir, REALM_ID = constants
+    message_id, usermessage_id = ids
+    json_names = os.listdir(slack_dir + '/' + channel)
+    users = json.load(open(slack_dir + '/users.json'))
+    zerver_message = []
+    zerver_usermessage = []
+
+    slack_usermention_regex = r"(<@)([a-zA-Z0-9]+)(\|)?([a-zA-Z0-9]+)?(>)"
+
+    # TODO mapping italics, bold, strikethrough etc
```

Comment:
A complete list to expand the "etc" would be clearer. I'm sure the list is finite.